src/main/webapp/index.html

```diff
@@ -1,19 +1,22 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <meta charset="UTF-8" content="width=device-width; height=device-height">
+    <title>Meme Generator</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="background-color:powderblue;">
```

Comment:
Just for later updates try to make the styles in the css (i know that is only one attribute)

Reply:
+1 I agree